src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -474,6 +483,9 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, OrgPersonMix
       // cannot remove proprietor/partner
       return false
     }
+    if (this.isLimitedConversionRestorationFiling || this.isLimitedExtendRestorationFiling) {
+      return true
+    }
```

Comment:
add extension and conversion to can remove org-person function

Reply:
PS Thanks for changing the section titlebar/style. I didn't notice that the screenshot doesn't match the uxpin, but I'm sure your new version is good 👍

Reply:
Here is the screenshot with changes:

![image](https://user-images.githubusercontent.com/116035339/224817665-a3e70171-2916-4113-854e-eb01a1a6c5ba.png)